README.md

```diff
@@ -347,6 +348,59 @@ TotalCurrentRevenue.calculate
 AverageDailyRevenue.result
 ```
 
+## Rule and Ruleset
+
+### When to use it
+
+Rule objects provide a place for conditional logic
+
+### Assumptions and rules
+
+* Rule has `#satisfied?`, `#applicable?` and `#forceable?` methods
+* Custom rule has to implement at least `#satisfied?` method
+* Rule requires a subject as first argument
+* Multiple rules can be joined into `Ruleset`
+
+### Examples
+
+#### Declaration
+
+```ruby
+class RoomIsAssignedRule < Rule
+  def satisfied?
+    subject.unit.present?
+  end
+
+  def forceable?
+    false
+  end
+end
+
+class RoomIsCleanRule < Rule
+  def satisfied?
+    !subject.unit.dirty?
+  end
+
+  def not_applicable?
+    !RoomIsAssignedRule.new(subject).satisfied?
+  end
+end
+
+RoomIsReadyRuleset = Class.new(Ruleset)
+RoomIsReadyRuleset.
+  add_rule(:room_is_assigned_rule).
+  add_rule(:room_is_clean_rule)
+```
+
+#### Usage
+
+```ruby
+RoomIsAssignedRule.new(room).satisfied?
+
+RoomIsReadyRuleset.new(room).satisfied?
+RoomIsReadyRuleset.new(room).satisfied?(force: true)
```

Comment:
Can you provide a description about how, when and why use this `force` flag? I don't think I understand how it works

Reply:
force flag will make rules that are "forceable" satisfied. Those which are not forceable will stay not satisifed

Reply:
That's how it works from our view, but what's the actual real-life use case? I can't think of any 😭

Reply:
@polakowski

Described in README now...